sample-manifests/spoke-sno/create_spoke_sno.sh

```diff
@@ -10,13 +10,13 @@ oc create -f 06_InfraEnv.yaml
 
 for i in {1..10}; do
 	sleep 5
-	ISO_URL=$(oc get infraenv ${CLUSTER_NAME} -o jsonpath={.status.isoDownloadURL})
-	if [[ ! -z ${ISO_URL} ]]; then
```

Comment:
proposed changing `-n` for `! -z` as suggested by [SC2236](https://github.com/koalaman/shellcheck/wiki/SC2236), but pls check what fits u better